packages/mermaid-mindmap/src/mindmap.spec.js

```diff
@@ -347,4 +347,16 @@ root
     expect(child.children.length).toEqual(2);
     expect(child.children[1].nodeId).toEqual('b');
   });
+  it('MMP-23 Rows with only spaces should not interfere', function () {
```

Comment:
In the test, add a blank line in the beginning of the Mermaid-code to be parsed and rendered